ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/RejectionReason.scala

```diff
@@ -50,6 +50,18 @@ object RejectionReason {
       "The maximum record time of the command exceeded"
   }
 
+  /** The ledger time of the transaction submission violated one of the
+    *  following constraints on ledger time:
+    *  - The difference between ledger time and the record time
+    *    in the ledger state at which the transaction was sequenced needs
+    *    to stay within bounds defined by the ledger.
+    *  - The ledger time has to be greater than or equal to the ledger time
+    *    of any contract referenced by the transaction.
+    */
+  final case class InvalidLedgerTime(reason: String) extends RejectionReason {
+    override def description: String = "Invalid ledger time: " + reason
```

Comment:
```suggestion
    override def description: String = s"Invalid ledger time: $reason"
```

Reply:
Is there any advantage to this syntax? I chose the `+` operator, as it is consistent with all other string concatenation operations in the same file.

Reply:
It's more 'scalaish' to use the string interpolation operator.

Reply:
It only has advantages in larger strings, of course favoring consistency make sense.